### ISSUE TITLE
v4 firefox responsive images not responsive in flexbox layouts.

### ISSUE THREAD

**Issue by LastRose** [twbs/bootstrap#17694]
images are going full width, rather than bounded by the grid - problem is in version 40-42 of firefox (that I'm aware of). 

ex. 

``` jade
.row
  .col-md-4
    img.img-responsive(src="http://placehold.it/600x300")
  .col-md-4
    img.img-responsive(src="http://placehold.it/600x300")
  .col-md-4
    img.img-responsive(src="http://placehold.it/600x300")
```

This can be fixed by setting `width: 100%` in .img-responsive, but not sure what that might break on other browsers.Turning off flexbox mode also fixes the issue, however would prefer to use flexbox.

**Comment by Turqueso** [comment]
shouldn't it be `img-fluid` instead of `img-responsive`? [docs](http://v4-alpha.getbootstrap.com/content/images/#responsive-images)

**Comment by jipexu** [comment]
should be !

**Comment by mdo** [comment]
Fixed by #19739 I believe.
